ehr/resources/schemas/dbscripts/sqlserver/a_ehr_lookups-17.20-17.21.sql

```diff
@@ -0,0 +1,3 @@
+/* File copied from EHR version upgrade to 17.21 in onprc17.2Prod branch */
+
+ALTER TABLE ehr_lookups.flag_categories ADD datedisabled DATETIME;
```

Comment:
This file isn't needed anymore. ONPRC is already on 19.1, and bootstrapping servers will run the rollup script.

Reply:
now removed